playbooks/collect_activity/collect_activity.yml

```diff
@@ -0,0 +1,37 @@
+---
+# A simple one-off playbook that gathers basic usage information from OpenedX hosts.
+# This is used by the `activity_csv` management command.
+
+- hosts: all
+  vars:
+    # This should be overridden to be more unique.
+    local_output_dir: /tmp/
+    remote_output_filename: /tmp/activity_report
+    section_name: '{{ inventory_hostname }}'
```

Comment:
@bdero Same comment about `section_name` here -- it's fairly generic, so if we need to set it (you could probably just reference `{{ inventory_hostname }}` directly in L21 below), it would be good to come up with a more specific name.